docker-scripts/load-sql.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# check input sql file exists
+SQL_FILE="$1"
+shift
+if [ ! -f $SQL_FILE ]; then
+    echo "Input file $SQL_FILE Not found"
+    exit
+fi
+if [ ! ${SQL_FILE: -4} == ".sql" ]; then
+    echo "Not a sql file: $SQL_FILE"
+    exit
+fi
+
+
+# default db name and table name
+x="${SQL_FILE##*/}"
+if [[ $x == $SQL_FILE ]]; then
+    DB_NAME=${SQL_FILE:0:$((${#SQL_FILE} - 4))}
+    # db name is the same name as the sql file
+else
+    x=${SQL_FILE:$((${#SQL_FILE} - ${#x}))}
+    DB_NAME=${x:0:$((${#x} - 4))}
+fi
+
+while [[ $# -gt 0 ]]
+do
+    key="$1"
+    case $key in
+        --dbname)
+            DB_NAME="$2"
+            shift
+            shift
+            ;;
+        *)
+            echo "Wrong argument name $key"
+            exit
+            ;;
+    esac
+done
+
+echo 'DB_NAME='$DB_NAME
+
+# create table and drop table
+docker exec -it kyrix_db_1 psql postgresql://postgres:kyrixftw@localhost/postgres -c "CREATE DATABASE ${DB_NAME}";
+
+# copy file to kyrix_db_1
+docker cp $SQL_FILE kyrix_db_1:/$SQL_FILE
+
+# import sql file content into the selected database
+docker exec -it kyrix_db_1 /bin/sh -c "psql postgresql://postgres:kyrixftw@localhost/${DB_NAME} < ${SQL_FILE}"
```

Comment:
fyi hardcoded password... instead pull from the environment, and if needed make this the default...

Reply:
This script assumes local environment. It feels bad if we run setup-kyrix-var in user's local environment. I'm thinking this password thing should be in the config file (of course, need to fix #64 first), so that every script that needs it can read from it.